schema-guide.md

```diff
@@ -336,13 +328,16 @@ licenses, it is assumed their relationship is OR, not AND.
 
 - **type**: A [`definitions.reference`](#definitionsreference) object.
 - **required**: `false`
-- **description**: ...
+- **description**: The reference that should be cited when citing the work, instead of using the software or dataset metadata in this CITATION.cff file.  
```

Comment:
```suggestion
- **description**: A reference to another work that should be cited instead of the software or dataset itself.  
```

Changed according to the proposed change in the schema.